app/src/main/java/com/phivle/fitnesstasks/SectionsPagerAdapter.java

```diff
@@ -0,0 +1,53 @@
+package com.phivle.fitnesstasks;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+
+import com.phivle.fitnesstasks.fragments.CalendarFragment;
+import com.phivle.fitnesstasks.fragments.SummaryFragment;
+import com.phivle.fitnesstasks.fragments.WorkoutFragment;
+
+/**
+ * Created by praniti on 10/10/17.
+ */
+
+public class SectionsPagerAdapter extends SmartFragmentStatePagerAdapter {
+    private static int NUM_ITEMS = 3;
+    private String tabTitles[] = new String[]{ "Calendar", "Workout", "Summary" };
```

Comment:
are we no longer calling it `Achievements` and instead using `Summary`

Reply:
I wasn't sure ...so thought would keep it open and you can change in your PR, but felt the new wireframe doesn't align with Achievements